src/message.rs

```diff
@@ -11,14 +12,46 @@ use fvm_shared::{
 use multihash_codetable::{Code, MultihashDigest as _};
 use serde::{Deserialize, Serialize};
 
-fn from_cbor_blake2b256<S: serde::ser::Serialize>(obj: &S) -> Result<Cid, Error> {
+fn from_cbor_blake2b256<S: serde::ser::Serialize>(
+    obj: &S,
+) -> Result<Cid, fvm_ipld_encoding::Error> {
     let bytes = fvm_ipld_encoding::to_vec(obj)?;
     Ok(Cid::new_v1(
         fvm_ipld_encoding::DAG_CBOR,
         Code::Blake2b256.digest(&bytes),
     ))
 }
 
+fn check_address_prefix(s: &str, n: Network) -> bool {
+    if s.len() == 0 || s.len() < 2 {
+        return false;
+    }
```

Comment:
The second alternative contains the first one.

```suggestion
    if s.len() < 2 {
        return false;
    }
```